Sources/SwiftParser/Patterns.swift

```diff
@@ -10,7 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-@_spi(RawSyntax) import SwiftSyntax
+@_spi(RawSyntax) @_spi(ExperimentalLanguageFeatures) import SwiftSyntax
```

Comment:
The `@_spi(ExperimentalLanguageFeatures)` is no longer necessary since you removed support for `_resultDependsOn`.

Reply:
I'll let it be because I am going to be using it in the follow on PR.